modules/ROOT/pages/afm-monitoring-private-apis.adoc

```diff
@@ -5,19 +5,60 @@ endif::[]
 :page-aliases: afm-delete-private-location.adoc
 :imagesdir: ../assets/images
 
-Private APIs are APIs with endpoints that are accessible only within your network. API Functional Monitoring (AFM) lets you monitor private APIs if you have a subscription to Anypoint Virtual Private Cloud (Anypoint VPC). With Anypoint VPC, you can run workers in CloudHub in a virtual, private, and isolated network segment, rather than in a region in which resources are shared.
+API Functional Monitoring (AFM) lets you monitor private APIs if you have a subscription to Anypoint Virtual Private Cloud (Anypoint VPC).
 
-A _private location_ is a worker that runs in a CloudHub environment that is associated with an instance of Anypoint VPC. AFM can create workers within Anypoint VPC that can run tests against APIs that are accessible only within the network that Anypoint VPC is configured for. The APIs must be deployed in CloudHub. You can create multiple private locations in a single instance of Anypoint VPC.
+== Definition of "Private API"
+Private APIs are APIs with endpoints that are accessible only within your network.
 
+== Definition of "Private Location"
+With Anypoint VPC, you can run workers in CloudHub in a virtual, private, and isolated network segment, rather than in a region in which resources are shared. A _private location_ is a worker that runs in a CloudHub environment that is associated with an instance of Anypoint VPC. AFM can create workers within Anypoint VPC that can run tests against APIs that are accessible only within the network that Anypoint VPC is configured for. The APIs must be deployed in CloudHub. You can create multiple private locations in a single instance of Anypoint VPC.
+
+[NOTE]
+====
+You can also test public APIs by using private locations. Public APIs are APIs with endpoints that are exposed to the open internet.
+
+Also, you can test private APIs from public locations if you whitelist the IP addresses of those locations. You might want to test a private API from a public location if you do not want to use vCores on testing private APIs or if you have no vCores available.
```

Comment:
I wonder if we can use "allow" or "include" or "add" or "specifically include" instead of "whitelist"? 
Just thinking out loud. I'll add these to the eval doc, I guess.

Reply:
@kbostrom , I can't use other terms until the Runtime Manager docs also use them.